src/aura/CMP_Acc_Naming_Dropdown/CMP_Acc_Naming_DropdownHelper.js

```diff
@@ -0,0 +1,82 @@
+({
+    init : function(component) {
+        //Retrieving account naming options
+        this.getAdminAccNameFormatOptions(component);
+        this.getHHAccNameFormatOptions(component);
+    },
+    onSelectChange : function(component) {
+        var selectedVal = component.find("nameFormatDropDown").get("v.value");
+        if (prefix === 'hed__') {
+            var accNamingOther = $A.get("$Label.hed.acctNamingOther");
+        } else {
+            var accNamingOther = $A.get("$Label.c.acctNamingOther");
+        }
+        if(selectedVal === accNamingOther) {
+            component.set("v.otherDisplay", true);
+        } else {
+            component.set("v.otherDisplay", false);
+        }
+        component.set("v.nameFormat", selectedVal);
+    },
+    
+    saveSetting : function(component) {        
+        //Get selected value
+        var selectedVal = component.find("nameFormatDropDown").get("v.value");
+        //Set the selected value in the settings (so it gets saved)
+        component.set("v.setting", selectedVal);
+        if (prefix === 'hed__') {
+            var accNamingOther = $A.get("$Label.hed.acctNamingOther");
+        } else {
+            var accNamingOther = $A.get("$Label.c.acctNamingOther");
+        }
+        if (selectedVal === ($A.get("$Label.c.acctNamingOther")))
+        {
+            selectedVal = component.find("otherText").get("v.value");
+            component.set("v.otherSetting", selectedVal);
+        }
+
+    },
+
+    getAdminAccNameFormatOptions : function(component) {
+        var adminAccNameFormatOptions = [];
+        if (prefix === 'hed__')
+        {
+            adminAccNameFormatOptions.push($A.get("$Label.hed.lastNameAdminAcc"));
+            adminAccNameFormatOptions.push($A.get("$Label.hed.firstNameLastNameAdminACC"));
+            adminAccNameFormatOptions.push($A.get("$Label.hed.lastNameFirstNameAdminAcc"));
+            adminAccNameFormatOptions.push($A.get("$Label.hed.acctNamingOther"));
+        }else{
+            adminAccNameFormatOptions.push($A.get("$Label.c.lastNameAdminAcc"));
+            adminAccNameFormatOptions.push($A.get("$Label.c.firstNameLastNameAdminACC"));
+            adminAccNameFormatOptions.push($A.get("$Label.c.lastNameFirstNameAdminAcc"));  
+            adminAccNameFormatOptions.push($A.get("$Label.c.acctNamingOther"));
+          
+        }
+        component.set("v.adminAccNameFormatOptions", adminAccNameFormatOptions);
+    },
+
+    getHHAccNameFormatOptions : function(component) {
+        var hhNameFormatOptions = [];
+        if (prefix === 'hed__')
+        {
+            hhNameFormatOptions.push($A.get("$Label.hed.None"));
+            hhNameFormatOptions.push($A.get("$Label.hed.lastNameHH"));
+            hhNameFormatOptions.push($A.get("$Label.hed.lastNameFirstNameHH"));
+            hhNameFormatOptions.push($A.get("$Label.hed.firstNameLastNameHH"));
```

Comment:
similar to the Admin account. The option string looks different.

Reply:
I think it makes sense. Although, I don't see the point of those parenthesis. Maybe a question for PM or Jace. I will ask them under the WI.